share/revng/test/tests/abi/gather_symbols.py

```diff
@@ -0,0 +1,398 @@
+import argparse
+import re
+import sys
+from contextlib import closing
+from dataclasses import dataclass
+from mmap import ACCESS_READ, mmap
+
+
+def accept_arguments():
+    parser = argparse.ArgumentParser(
+        description="A quick disassembly analyzer that emits some addresses as a C header.",
```

Comment:
1. What is the input of this script? binutils' `objdump`? `llvm-objdump`?
2. Does this work with PE/COFF too?

Reply:
Done.